src/orca/Orca.java

```diff
@@ -0,0 +1,267 @@
+        System.out.println(LOGO);
+        System.out.println("    Hello! I'm Orca, your assistant chatbot.");
+        System.out.println("    What can I do for you?");
+        System.out.println("    --------------------------------------------------\n");
+    }
+
+    public static void printByeMessage() {
+        System.out.println("    --------------------------------------------------");
+        System.out.println("    Bye. Hope to see you again soon!");
+        System.out.println("    --------------------------------------------------\n");
+    }
+
+    public static void printUnknownCommandMessage() {
+        System.out.println("    --------------------------------------------------");
+        System.out.println("    Sorry, I don't understand what you mean.");
+        System.out.println("    --------------------------------------------------\n");
+    }
+
+    public static void printTasks() {
+        System.out.println("    --------------------------------------------------");
+        System.out.println("    Here are the tasks in your list:");
+        for (int i = 0; i < tasks.size(); i++) {
+            System.out.println("    " + (i + 1) + "." + tasks.get(i));
+        }
+        System.out.println("    --------------------------------------------------\n");
+    }
+
+    public static void printLatestAddedTask() {
+        System.out.println("    --------------------------------------------------");
+        System.out.println("    Got it. I've added this task:");
+        System.out.println("      " + tasks.get(tasks.size() - 1));
+        System.out.println("    Now you have " + tasks.size() + " tasks in the list.");
+        System.out.println("    --------------------------------------------------\n");
+    }
+
+    public static void findCommandType() {
+        if (userInput.equals("bye")) {
+            commandType = CommandType.BYE;
+        } else if (userInput.equals("list")) {
+            commandType = CommandType.LIST;
+        } else if (userInput.startsWith("mark")) {
+            commandType = CommandType.MARK;
+        } else if (userInput.startsWith("unmark")) {
+            commandType = CommandType.UNMARK;
+        } else if (userInput.startsWith("todo")) {
+            commandType = CommandType.TODO;
+        } else if (userInput.startsWith("deadline")) {
+            commandType = CommandType.DEADLINE;
+        } else if (userInput.startsWith("event")) {
+            commandType = CommandType.EVENT;
+        } else if (userInput.startsWith("delete")) {
+            commandType = CommandType.DELETE;
+        } else {
+            commandType = CommandType.UNKNOWN;
+        }
+    }
+
+    public static boolean isInputAvailable() {
+        return in.hasNextLine();
+    }
+
+    public static void readUserInput() {
+        userInput = in.nextLine();
+    }
+
+    public static void markTask(int taskNo) throws OrcaException {
```

Comment:
Instead of `taskNo`, maybe you can consider renaming it as `taskNumber` so that it is more readable.